make/RunTests.gmk

```diff
@@ -419,38 +438,22 @@ ifeq ($(TEST), )
   $(error Cannot continue)
 endif
 
+define ParseTestSelection
+$(strip $(foreach parser,ParseCustomTestSelection ParseGtestTestSelection ParseMicroTestSelection ParseJtregTestSelection ParseSpecialTestSelection \
+ ,$(call $(parser),$1)))
+endef
```

Comment:
As this is logically a one-liner macro, please format according to guidelines. Also try to keep lines shortish. Something like this:

```suggestion
ParseTestSelection = \
  $(strip $(foreach parser, ParseCustomTestSelection ParseGtestTestSelection ParseMicroTestSelection \ 
      ParseJtregTestSelection ParseSpecialTestSelection, \
    $(call $(parser), $1) \
  ))
```

Reply:
I made it a one-liner, and I refactored the code to use `or`